projects/doc/README.rst

```diff
@@ -25,7 +25,7 @@ ircs://irc.libera.chat/#gentoo-haskell to get help.
 
 .. _TODO list: TODO.rst
 .. _bug reports: http://tinyurl.com/2l3p48
-.. _some git basics: http://progit.org/book/
+.. _some git basics: https://git-scm.com/book/en/v2 
```

Comment:
It looks like your commit from the other PR snuck in here somehow.

Reply:
I have made a change to the commit message, and also reverted the other change in my master branch. The problem arose because I made the first pull request, then found the issue with the book link and made the change in the same branch. I guess github automatically added that change when I pushed it. I need to be more disciplined in working in a separate branch for each change - as I did in the end with the book link.

Reply:
Hey, no worries! A lot of these things just take practice.

Reply:
A useful command you might not know about is `git cherry-pick`, which can copy a single commit from one branch to another, which would let you make commits in any branch and shuffle them around later (removing extras with `git rebase`).